test/container/Dockerfile

```diff
@@ -56,7 +56,7 @@ ENV GOPATH=/go
 COPY hack/install.sh hack/tool-versions.sh go.* ./
 COPY hack/installers installers
 RUN mkdir -p gitops-engine
```

Comment:
This is an NIT, and i realize it's in the main Dockerfile as well, but `COPY` created the destination folders. 

https://docs.docker.com/reference/dockerfile/#destination-1

do we need that `mkdir`?